### PR TITLE
Adding insertListVoid which passes noResult as true

### PR DESCRIPTION
**Pull request by TheKewlStore**
See https://github.com/vitusortner/floor/issues/350

### CODE REVIEW

File: floor/lib/src/adapter/insertion_adapter.dart [hunk 1]
```diff
@@ -87,4 +87,24 @@ class InsertionAdapter<T> {
     }
     return result;
   }
+  
+    Future<void> _insertListVoid(
+      final List<T> items,
+      final ConflictAlgorithm conflictAlgorithm,
+      ) async {
```
Comment (vitusortner) [inline floor/lib/src/adapter/insertion_adapter.dart:94]: 🔍 Formatting seems to be off here
```suggestion
  Future<void> _insertListVoid(
    final List<T> items,
    final ConflictAlgorithm conflictAlgorithm,
  ) async {
```

File: floor/lib/src/adapter/insertion_adapter.dart [hunk 2]
```diff
@@ -36,7 +36,7 @@ class InsertionAdapter<T> {
     final OnConflictStrategy onConflictStrategy,
   ) async {
     if (items.isEmpty) return;
-    await _insertList(items, onConflictStrategy);
+    await _insertListVoid(items, onConflictStrategy);
```
Comment (vitusortner) [inline floor/lib/src/adapter/insertion_adapter.dart:39]: 💡 You could inline the functionality as it's only used once. You also need to make sure to map `OnConflictStrategy` to sqflite's `ConflictAlgorithm`.

```suggestion
    final batch = _database.batch();
    for (final item in items) {
      batch.insert(
        _entityName,
        _valueMapper(item),
        conflictAlgorithm: onConflictStrategy.asSqfliteConflictAlgorithm(),
      );
    }

    await batch.commit(noResult: true);

    if (_changeListener != null) {
      _changeListener.add(_entityName);
    }
  }
```

**Comment by TheKewlStore** [comment]
Hello, sorry for the delay in getting back to you on this! I accepted the changes you suggested on inlining the function and the formatting issues. I also adjusted the test you mentioned (by ensuring that it checks for the noResult parameter to be true). Let me know if there are any other changes necessary!

**Comment by vitusortner** [comment]
There was a bracket too much, which I got rid of. Thanks 🙏

**Pull request merged**